src/lambda/config/templates.ts

```diff
@@ -53,31 +72,56 @@ export class DefaultLoadTemplatesConfigContext implements LoadTemplatesConfigCon
     }
 }
 
+export function getTemplatesConfigPath(workspaceFolderPath: string): string {
+    return _path.join(
+        workspaceFolderPath,
+        '.aws',
+        'templates.json'
+    )
+}
+
+export function getTemplateRelativePath(
```

Comment:
FYI, the PR that I'm about to merge contains a new function `getNormalizedRelativePath`--once it's merged, switch to that instead.

Reply:
It's merged--check out `src/shared/utilities/pathUtilities.ts`.